app/scripts/metamask-controller.js

```diff
@@ -1472,10 +1472,14 @@ export default class MetamaskController extends EventEmitter {
       getMetaMetricsState: () =>
         this.metaMetricsController.state.participateInMetaMetrics ?? false,
       state: initState.UserStorageController,
+      env: {
+        isAccountSyncingEnabled: true,
+      },
       messenger: this.controllerMessenger.getRestricted({
         name: 'UserStorageController',
         allowedActions: [
           'KeyringController:getState',
+          'KeyringController:addNewAccount',
```

Comment:
Nice addition to the allowed actions and events.

I'm thinking if it is worth adding comments to indictate which actions/events are for what features.

Mostly because this controller will start to have a large set of actions & events it will listen to with all the syncing features.

Meh, we can do this later.